napari/layers/shapes/_shapes_constants.py

```diff
@@ -35,6 +35,22 @@ class Mode(StringEnum):
     VERTEX_REMOVE = auto()
 
 
+class ColorMode(StringEnum):
```

Comment:
Not necessarily needed for this PR, but I think clearly there should only be one ColorMode enum shared by all the layers that need it!